src/main/java/com/spotinst/sdkjava/model/api/ocean/ecs/ApiClusterHeadroomsSpecification.java

```diff
@@ -0,0 +1,75 @@
+package com.spotinst.sdkjava.model.api.ocean.ecs;
+
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.spotinst.sdkjava.client.rest.IPartialUpdateEntity;
+
+import java.util.HashSet;
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonFilter("PartialUpdateEntityFilter")
+public class ApiClusterHeadroomsSpecification implements IPartialUpdateEntity {
```

Comment:
Should we remove 'Cluster' from this file, since it is used at both cluster and launchspec level. Also, please add a comment in the file that this class is reused.

Reply:
Leaving it as cluster level and used in launch specification. Added comment